### DISCUSSION TITLE
Do picture extraction tools for the original GB Camera also work with this flashcart?

### DISCUSSION THREAD

**Discussion by KuestenKeks** [HDR/Gameboy-Camera-Flashcart#5] [Q&A] [answered]
Do tools like the like the GBxCart RW GB Camera Saver work with the flashcart?

I suppose they should work if an original GB Camera ROM is flashed? And for other ROMs it depends on the save format being compatible?

So I guess picture extraction for the ROM [2bit-pxlr-studio-next](https://github.com/untoxa/2bit-pxlr-studio-next) should work, since their save format is "retro-compatible"?

> The save format used here is retro-compatible with any version of the Game Boy Camera rom. This means that you can switch between roms while keeping the same save without loosing your precious images and data.

Cheers! :)

**Comment by HDR** [comment, marked as answer]
The flashcart is functionally the same as an original one in that regard, meaning all tools made for the original game boy camera will work, 2bit pxlr studio (next) should work as well as the save structure is the same.
